spy/analyze/scope.py

```diff
@@ -285,6 +285,8 @@ def declare_VarDef(self, vardef: ast.VarDef) -> None:
             vardef.loc,
             vardef.type.loc,
         )
+        if vardef.value is not None:
+            self.declare(vardef.value)
```

Comment:
when I read this line I though "this should have a test in test_scope"... and then I saw it's there! Good job :muscle: